packages/faustwp-cli/utils/shouldFireTelemetryEvent.ts

```diff
@@ -1,11 +1,17 @@
 import { userConfig } from './userConfig.js';
+import { getWpSecret } from '../../faustwp-core/src/lib/getWpSecret.js';
 
 export function shouldFireTelemetryEvent(): boolean {
-  const hasSecretKey = process.env.FAUSTWP_SECRET_KEY;
+  const args = getCliArgs();
+  const arg1 = args[0];
   const hasAnonymousId = userConfig.get('telemetry.anonymousId');
   const hasTelemetryEnabled = userConfig.get('telemetry.enabled') === true;
 
-  const shouldFireEvent = hasSecretKey && hasAnonymousId && hasTelemetryEnabled;
+  const shouldFireEvent =
+    getWpSecret() &&
+    hasAnonymousId &&
+    hasTelemetryEnabled &&
+    runningUsingProperCommands;
```

Comment:
```suggestion
    getWpSecret() &&
    hasAnonymousId &&
    hasTelemetryEnabled;
```

Same as above, remnant of a bad merge.